version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.18.7-SNAPSHOT"
+ThisBuild / version := "0.18.7"
```

Comment:
I don't think we need to change the version here. This will be automatically updated with the release of a new version

Reply:
@dimamo5 yeap, you're right! Reverted the version updating commit, should be good now 👍